apps/coscrad-frontend/src/utils/generic-components/presenters/multilingual-text-presenter.tsx

```diff
@@ -22,27 +22,30 @@ export const MultilingualTextPresenter = ({
     const textItemWithDefaultLanguage =
         items?.find((item) => item.languageCode === defaultLanguageCode) || null;
 
+    const mainText = isNullOrUndefined(textItemWithDefaultLanguage)
+        ? items?.find((item) => item.role === MultilingualTextItemRole.original)
+        : textItemWithDefaultLanguage;
+
     const translations = items?.filter((items) => items.languageCode !== defaultLanguageCode);
 
     return (
         <Box width={'fit-content'} data-testid="multilingual-text-display">
             <Accordion elevation={0}>
-                <AccordionSummary expandIcon={<ExpandMoreIcon />}>
+                <AccordionSummary
+                    expandIcon={<ExpandMoreIcon />}
+                    data-testid="multilingual-text-main-text-item"
+                >
                     <Typography variant="h4" margin={'auto 0'}>
-                        {isNullOrUndefined(textItemWithDefaultLanguage)
-                            ? 'Translations'
-                            : textItemWithDefaultLanguage.text}
-                        {isNullOrUndefined(textItemWithDefaultLanguage) ? null : (
-                            <Tooltip
-                                title={`${getLabelForLanguage(
-                                    textItemWithDefaultLanguage.languageCode
-                                )}, '${textItemWithDefaultLanguage.role}'`}
-                            >
-                                <IconButton>
-                                    <LanguageIcon />
-                                </IconButton>
-                            </Tooltip>
-                        )}
+                        {mainText.text}
+                        <Tooltip
```

Comment:
It feels like you should have a `MultilingualTextItemPresenter` that takes in the entire multilingual text item and deals with the text and language code. Maybe there's a reason not to do this?

Reply:
Yes, i was wondering about that.  I could give it a go.

Reply:
There is already the `TranslatedLanguageTextPresenter` for the translations, so this `MultilingualTextItemPresenter` would only be applied to the `primaryMultilingualTextItem`.  The main difference between the two is that the primary text has different formatting (H4, margin bottom) than the translations, so a generic presenter would have to account for that somehow (passed in configuration?).

Reply:
I'd prefer to pass in the styling to customize a single source of truth presenter if that makes sense. But if it's awkward, don't worry about it.